### DISCUSSION TITLE
How to move backup from server to server and restore this

### DISCUSSION THREAD

**Discussion by megatonix** [nextcloud/all-in-one#2904] [Questions] [answered]
Like the Title, i have backup on my  1server but i just lunch new nextcloud on 2server and i wanna restore backup from 1server to 2server. How exacly can i do it? I already know how to mount backup to tmp/borg and i see backups by date but how can i "mount" specific backup to another instance?

**Comment by szaimen** [comment, marked as answer]
You simply transfer the whole backup archive to the 2nd server, start a new AIO instance and instead of entering a domain in the AIO interface, you enter the secret of the borg archive and the location of it on the server.

**Reply by megatonix** [reply]
Ok but i don't want all my backups, i need only one from 1 day. Why cloning 10 backups in whole dir?
I can decript whole borg dir and move 1 backup from eg. 15.06.2023 but when i clone him to other server i cannot point him in my new instance cuz this backup is nonecrypted

**Reply by szaimen** [reply]
Since borg uses compression, deduplication and further methods to reduce the size, will the whole archive not be much bigger than 1 backup.
